content/en/get-started/overview/index.md

```diff
@@ -55,10 +55,11 @@ The table below lists the recommended wallet implementations:
 
 | Name                                             |                                                                                                                                                                                             |
 | ------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [Lotus](https://lotus.filecoin.io)                         | Lotus can manage bls, sec1p256k1 wallets and supports [Ledger integration](https://lotus.filecoin.io/docs/set-up/manage-fil/#ledger).                                                                                                |
-| [Glif Safe](https://wallet.glif.io/) | Glif is a lightweight web interface to send and receive Filecoin with a Ledger device ([instructions](https://reading.supply/@glif/install-the-filecoin-app-on-your-ledger-device-y33vhX)). Check out the [Glif Safe GitHub repo](https://github.com/glifio/safe) for more information.|
+| [Lotus](https://lotus.filecoin.io)                         | Lotus can manage bls, sec1p256k1 wallets and supports [Ledger integration](https://lotus.filecoin.io/docs/set-up/manage-fil/#ledger).|
+| [Glif Safe](https://safe.glif.io/) | Glif Safe is a multisig wallet with a lightweight web interface to send and receive Filecoin with a Ledger device ([instructions](https://reading.supply/@glif/install-the-filecoin-app-on-your-ledger-device-y33vhX)). Check out the [Glif Safe GitHub repo](https://github.com/glifio/safe) for more information.|
+| [Glif Wallet](https://wallet.glif.io/) | Glif Wallet is a lightweight web interface to send and receive Filecoin with a Ledger device ([instructions](https://reading.supply/@glif/install-the-filecoin-app-on-your-ledger-device-y33vhX)).|
 
-Both Lotus and the Glif wallet support [Ledger](https://www.ledger.com/) hardware devices, allowing you to use Filecoin without ever storing your private keys on a network-connected device. This can help protect your valuable private keys from malicious software on your computer and so is commonly used for accounts with large balances.
+Lotus, Glif Safe, and Glif Wallet all support [Ledger](https://www.ledger.com/) hardware devices, allowing you to use Filecoin without ever storing your private keys on a network-connected device. This can help protect your valuable private keys from malicious software on your computer and so is commonly used for accounts with large balances.
```

Comment:
```suggestion
Lotus, Glif Safe, and Glif Wallet all support [Ledger](https://www.ledger.com/) hardware devices, allowing you to use Filecoin without storing your private keys on a network-connected device. This can help protect your valuable private keys from malicious software on your computer and is commonly used for accounts with large balances.
```